eth_tester/utils/throws_contract.py

```diff
@@ -29,46 +40,78 @@
     "080fd5b606060a8565b005b3415606c57600080fd5b607260b6565b60405180828152602001915050"
     "60405180910390f35b3415609257600080fd5b60a6600480803590602001909190505060bc565b005"
     "b6000151560b457600080fd5b565b60005481565b80600081905550505600a165627a7a72305820d5"
-    "8a6b595eeb1a765e71924eec6e96c98c1fe9e90876ec232593b9ebb9c686500029"
+    "8a6b595eeb1a765e71924eec6e96c98c1fe9e90876ec232593b9ebb9c686500029",
```

Comment:
I'd rather put the new bytecode in a new variable.  Something like `REVERT_BYTECODE`